misc/simulators/hashbasedsig.py

```diff
@@ -146,7 +146,7 @@ def calculate(K, K1, K2, q_max, L_hash, trees):
             lg_q = lg(q_cand)
             lg_pforge = [lg_px[x] + (lg_q*x - lg_K2)*q_cand for x in xrange(1, j)]
             if max(lg_pforge) < -L_hash + lg(j) and lg_px[j-1] + 1.0 < -L_hash:
-                #print "K = %d, K1 = %d, K2 = %d, L_hash = %d, lg_K2 = %.3f, q = %d, lg_pforge_1 = %.3f, lg_pforge_2 = %.3f, lg_pforge_3 = %.3f" \
+                #print("K = %d, K1 = %d, K2 = %d, L_hash = %d, lg_K2 = %.3f, q = %d, lg_pforge_1 = %.3f, lg_pforge_2 = %.3f, lg_pforge_3 = %.3f" \)
```

Comment:
Whatever tool you used to do this get the backslash case wrong.

Reply:
Thanks. I did a textual replacement then visually inspected, but not rigorously.

Reply:
Fixed in 5a40bf4 and validated there aren't other cases of a line terminating in a `\` following a commented print.